submissions/Ogenna-Ogbogu/about me.txt

```diff
@@ -0,0 +1,5 @@
+Hello world!
```

Comment:
Ensure that this file is in the right folder. Read the task's instructions carefully and ensure you have satisfied all requirements.